R/utilities-file.R

```diff
@@ -21,7 +21,14 @@
 
 # Adds the paramter pathToAdd to the system path
 .addPathToSystemPath <- function(pathToAdd) {
-  if (file.exists(pathToAdd)) {
-    Sys.setenv(path = paste(pathToAdd, Sys.getenv("path"), sep = ";"))
+  if (!file.exists(pathToAdd)) {
+    return()
   }
+
+  # Adding path only supported on windows
+  if (.Platform$OS.type != "windows") {

```

Comment:
@Yuri05 Removing this call in Linux as this does not work

Also https://stackoverflow.com/questions/41494585/setting-ld-library-path-from-inside-r

Setting hte path under linux from R won't work
So the path to the lib must be in LD_LIBRARY_PATH or must be set before starting R